frontend/src/devicelist/ContentsTable.jsx

```diff
@@ -50,23 +48,31 @@ export default class ContentsTable extends React.Component {
     const { classes } = this.props;
     return (
       <>
-        <Box style={{ display: "flex", flexGrow: 1, margin: "1em 0em" }}>
-          <SearchBar />
-          <Box className="alignRightFloat">
-            <DeviceListSortSelector />
-          </Box>
-        </Box>
-        <Box style={{ display: "flex", flexGrow: 1, maxHeight: 500 }}>
+        <Box
+          style={{
+            display: "flex",
+            flexGrow: 1,
+            maxHeight: 500
+          }}
+          border={1}
+          borderColor="#f1f1f1"
+        >
           <VerticalTabs
             value={value}
             setValue={this.handleValueChange}
             classes={classes}
           />
-          <TabPanel value={value} index={0}>
-            <DevicesTable devices={senders} />
+          <TabPanel value={this.state.value} index={0}>
```

Comment:
Use the `value` const variable declared on line 47. Same for when you use `this.state.senders` or `this.state.receivers`. This will eliminate some code smells in this file.